dbt/include/databricks/profile_template.yml

```diff
@@ -8,6 +8,12 @@ prompts:
   token:
     hint: 'dapiXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
     hide_input: true
+  _choose_unity_catalog:
+    'use Unity Catalog':
+      catalog:
```

Comment:
What if the user leaves it empty? Will it use current_catalog()?

Reply:
The prompt will ask again:

```
Desired unity catalog option (enter a number): 1
catalog (initial catalog):
catalog (initial catalog):
catalog (initial catalog):
catalog (initial catalog):
catalog (initial catalog):
```

If the user explicitly set it empty by editing the profile file, dbt command will fail:

```
  Credentials in profile "xxx", target "dev" invalid: Runtime Error
    Invalid catalog name : .
```